src/hid_freebsd.c

```diff
@@ -114,7 +178,24 @@ fido_hid_manifest(fido_dev_info_t *devlist, size_t ilen, size_t *olen)
 
 	for (i = *olen = 0; i < MAX_UHID && *olen < ilen; i++) {
 		snprintf(path, sizeof(path), "/dev/uhid%zu", i);
-		if (copy_info(&devlist[*olen], path) == 0) {
+		if (copy_info_uhid(&devlist[*olen], path) == 0) {
+			devlist[*olen].io = (fido_dev_io_t) {
+				fido_hid_open,
+				fido_hid_close,
+				fido_hid_read,
+				fido_hid_write,
+			};
+			++(*olen);
+		}
+	}
+
+#if defined(HAS_HIDRAW)
+	if (*olen > 0)
+		return (FIDO_OK);
+
+	for (i = *olen = 0; i < MAX_UHID && *olen < ilen; i++) {
+		snprintf(path, sizeof(path), "/dev/hidraw%zu", i);
+		if (copy_info_hidraw(&devlist[*olen], path) == 0) {
```

Comment:
Please excuse my ignorance, but the relationship between `uhid(4)` and `hidraw(4)` in FreeBSD 13 is not completely clear to me. Is it the case that, if `hidraw(4)` is enabled, then `uhid(4)` will no longer attach to HID devices?